### DISCUSSION TITLE
Floating Tags moving from buster to bullseye?

### DISCUSSION THREAD

**Discussion by chezsmithy** [dotnet/dotnet-docker#4056] [General]
Is there a plan to move the floating tags from buster to bullseye due to buster becoming end of life this month?

I'm also curious on what the plan is for the floating tags in the future?

**Comment by mthalman** [comment]
I haven't seen anything official about EOL for Buster. But even when it does become EOL, support for it will transition to the [Debian LTS team](https://www.debian.org/lts/) where it will still receive security updates. The only version of .NET that has supported container images for Buster is .NET Core 3.1 which is EOL in December of this year.
